kokoro/release/python/linux/config.sh

```diff
@@ -6,6 +6,12 @@ function pre_build {
     # Runs in the root directory of this repository.
     pushd protobuf
 
+    if [ -n "$IS_OSX" ]; then
```

Comment:
We probably should file a bug to multibuild and kokoro to let them include these.

Reply:
But libatomic is something that we include recently. I assume not everyone need it, right?